src/presentational-components/shared/content-list-empty-state.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import {
+  Bullseye,
+  Button,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+  Text,
+  TextContent,
+  TextVariants
+} from '@patternfly/react-core';
+
+const ContentListEmptyState = ({
```

Comment:
This component is exactly the same as `content-gallery-empty-state`. What's the reason behind this?

Reply:
There are UX mocks coming for each of these empty pages.

Reply:
So, please use the old component until you get the mocks. There is no reason to prepare files for the future.